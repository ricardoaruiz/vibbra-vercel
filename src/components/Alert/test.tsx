import { renderWithThemeProvider } from 'utils/test'

import { Alert } from '.'

describe('<Alert />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(
      <Alert message="Altert message" />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})
