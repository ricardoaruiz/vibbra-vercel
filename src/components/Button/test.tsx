import { renderWithThemeProvider } from 'utils/test'
import { Button } from '.'

describe('<Button />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Button />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
