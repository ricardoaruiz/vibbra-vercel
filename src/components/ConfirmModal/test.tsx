import { renderWithThemeProvider } from 'utils/test'

import { ConfirmModal } from '.'

describe('<ConfirmModal />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<ConfirmModal />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
