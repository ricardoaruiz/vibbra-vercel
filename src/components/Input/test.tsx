import { renderWithThemeProvider } from 'utils/test'

import { Input } from '.'

describe('<Input />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Input />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
