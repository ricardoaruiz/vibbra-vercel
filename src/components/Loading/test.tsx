import { renderWithThemeProvider } from 'utils/test'

import { Loading } from '.'

describe('<Loading />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Loading />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
