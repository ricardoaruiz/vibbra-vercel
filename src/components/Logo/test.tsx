import { renderWithThemeProvider } from 'utils/test'

import { Logo } from '.'

describe('<Logo />', () => {
  it('should render correctly', () => {
    const { container } = renderWithThemeProvider(<Logo />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
