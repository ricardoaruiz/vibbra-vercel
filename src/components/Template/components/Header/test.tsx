import { renderWithThemeProvider } from 'utils/test'

import { Header } from '.'

describe('<Header />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Header />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
