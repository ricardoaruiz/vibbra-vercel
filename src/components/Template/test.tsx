import { renderWithThemeProvider } from 'utils/test'

import { Template } from '.'

jest.mock('next/router', () => ({
  useRouter: () => ({
    push: jest.fn()
  })
}))

describe('<Template />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Template />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
