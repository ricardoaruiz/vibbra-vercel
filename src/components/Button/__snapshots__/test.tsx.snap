// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render the heading 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: 1px solid #6be7fe;
  border-radius: 0.5rem;
  padding: 1rem 0.5rem;
  outline: none;
  background-color: #6be7fe;
  color: #FAFAFA;
  font-size: 1.625rem;
  font-weight: 600;
  cursor: pointer;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
}

.c0:hover {
  background-color: #1fdbfd;
}

.c0:active {
  background-color: #01aecf;
}

.c0:focus {
  border: 1px solid #26AD9E;
}

.c0:disabled {
  background-color: #b7f3ff;
  cursor: default;
  pointer-events: none;
}

<button
  class="c0"
/>
`
